tests/integration/run_jsonrpc_commands.py

```diff
@@ -14,6 +14,8 @@ def run_shell_command(command: str) -> int:
     response = json.loads(process.stdout)
     if "error" in response:
```

Comment:
As you've just done below in `else` clause, you could add here `print("KO: ", command_and_args)` before exit

Reply:
ok